docs/docs.nix

```diff
@@ -79,6 +79,37 @@
             '';
           };
         };
+        generate-rust-docs = {
```

Comment:
this needs to be done in a derivation, such that it will be run in CI and ensure that docs are always up to date: https://crane.dev/API.html#cranelibcargodoc